pkg/block/s3/main_test.go

```diff
@@ -0,0 +1,86 @@
+package s3_test
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"os"
+	"testing"
+
+	"github.com/minio/minio-go/v7"
+	"github.com/minio/minio-go/v7/pkg/credentials"
+	"github.com/ory/dockertest/v3"
+)
+
+const (
+	minioContainerTimeoutSeconds = 10 * 60 // 10 min
+	bucketName                   = "bucket1"
+	minioTestEndpoint            = "127.0.0.1"
+	minioTestAccessKeyID         = "Q3AM3UQ867SPQQA43P2F"
+	minioTestSecretAccessKey     = "zuf+tfteSlswRu7BJ86wekitnifILbZam1KYY3TG"
+)
+
+var (
+	blockURL string
+	pool     *dockertest.Pool
+)
+
+func newClient(port string) *minio.Client {
```

Comment:
Return the `err` and in the test main func choose one of `panic` to `log.Fatal` on error.